paper.tex

```diff
@@ -194,18 +266,41 @@ \subsection*{Unambiguous low-level representation}
 \bottomrule
 \end{tabular}
 \end{center}
-\caption{Software support for Demes. We have included software infrastructure developed
+\caption{\label{tab-software}
+Software support for Demes. We have included software infrastructure developed
 for working with Demes models (parsing, validation, visualisation, etc)
 as well as all known downstream software that implements the specification,
 at the time of writing.}
 \end{table}
 
-\subsection*{Other stuff...}
+\section*{Discussion}
+
+This is an unstructured list of things we should mention:
 
 \begin{itemize}
-    \item General framework for inference (have been working on that over in \moments,
-        and perhaps it could be generalized to work with any simulation engine
-    \item Defining distributions of parameter values, perhaps?
+    \item Recent developments in software engineering have favoured
+    static, declarative configuration formats. It feels like a good idea to
+    have a Turing complete configuration format, but it turns out not to be.
+
+    \item Drawing parameters from distributions. Why we are static.
+    % https://github.com/popsim-consortium/demes-paper/issues/3
+
+    \item Why we've avoided turning this into a model of everything, allowing
+    for the specification of genome properties like recombination rates,
+    etc.
+
+    \item
+    % JK: I thought there was another simulation frontend somewhere..
```

Comment:
stdpopsim?

Reply:
I suppose it is, but that's not what I'm getting at. I thought there was some other effort at some point to make a single frontend for a bunch of simulators to simplify running simulations. The point I want to make is that this is the wrong way to go - we shouldn't be writing code to resolve the differences between the simulators and to try and abstract their models of demography. We should all agree on a model of demography, and get the individual simulators to implement it.